### ISSUE TITLE
Error on app opening

### ISSUE THREAD

**Issue by FBernuy** [alexcorvi/apexo#39]
Hi, excellent work here, it is an amazing project!

I am trying to setup a local server on win10. I have the coachDB running and was able to log into the app but when a tried to make a new patient with a new appointment, the app "crashed" and now I can't log again. Here is the error when I open the app. 

```
TypeError: Cannot read property 'type' of undefined
    at i.createElement.i.createElement.i.createElement.i.createElement.i.createElement.i.createElement.todayAppointments.map.e (https://web.apexo.app/app.js:31:54554)
    at Array.map (<anonymous>)
    at f.render (https://web.apexo.app/app.js:31:54417)
    at X (https://web.apexo.app/app.js:23:6498)
    at https://web.apexo.app/app.js:23:60091
    at he (https://web.apexo.app/app.js:23:12508)
    at e.track (https://web.apexo.app/app.js:23:16955)
    at f.n [as render] (https://web.apexo.app/app.js:23:60004)
    at f.<anonymous> (https://web.apexo.app/app.js:23:60865)
    at f.J.h.render (https://web.apexo.app/app.js:23:60877)
```

![image](https://user-images.githubusercontent.com/13875097/53689526-018ca180-3d37-11e9-92a3-ca00db5ff288.png)

Tried deleting the app and running it again, but it keeps throwing the same error msg.

What did a do wrong, and how can I fix it?

Thanks!

**Comment by FBernuy** [comment]
apparently, the problem started when I created an appointment while no treatment was created. Fixed it by creating a treatment in the DB and adding it to the appointment. I think you shouldn't be able to make appointments if there is no treatments.

**Comment by alexcorvi** [comment]
Fixed in v2.2.3, the application should automatically update to the latest version.
